docs/docs-next/docs/concepts/automation.md

```diff
@@ -1 +1,66 @@
-# Automation
\ No newline at end of file
+---
+title: About Automation
+---
+
+Dagster has support for several types of automation. All automation in Dagster responds to some external event.
+
+The first system, and the most basic, is the [Schedule](/guides/automation/schedules), which responds to time.
+
+[Sensors](/guides/automation/sensors) are like schedules, but they respond to an external event defined by the user.
+
+[Asset Sensors](/guides/automation/asset-sensors) are a special case of sensor that responds to changes in asset materialization
+as reported by the Event Log.
+
+Finally, the Declarative Automation system is a
+more complex system that uses conditions on the assets to determine when to execute.
+
+## About Schedules
```

Comment:
I wonder if we really need a 'Schedules' concept page or if Automation is enough?
We could then link to <concept/automation#Schedules>

Reply:
It might be fine for Schedules - it is something that everyone is probably familiar with. Do you think it will be a weird experience if Schedules doesn't have one, but something like Sensors does?